### PR TITLE
Show address that publishes a package

### PR DESCRIPTION
**Pull request by Jibz1**
Screen Shot 2022-06-24 at 10 48 21 AM

### CODE REVIEW

**Comment by bwann52** [comment]
I think calling it Publisher is more clear

File: explorer/client/src/pages/object-result/ObjectResult.tsx [hunk 1]
```diff
@@ -39,15 +43,44 @@ const Fail = ({ objID }: { objID: string | undefined }): JSX.Element => {
...
+                    .getTransactionWithEffects(resp.data.tx_digest)
+                    .then((txEff: TransactionEffectsResponse) => ({
+                        ...resp,
+                        publisherAddress: getTransactionSender(
+                            txEff.certificate
+                        ),
+                    }))
+                    .catch((err) => {
+                        console.log(err);
+                        // TODO: Not sure if I should show Genesis as Package Publisher or ignore it
```
Comment (Andrew47) [inline explorer/client/src/pages/object-result/ObjectResult.tsx:63]: I like the overall approach; it's efficient. 

Regarding packages such as 0x2 that exist at Genesis, perhaps the best approach would be to remove the row detailing the `Last Transaction ID` and input 'Genesis' as a string next to `Publisher`? Would this be a better approach, Bauer (@bwann52)? 

Below is a screenshot of what this PR delivers for package 0x2:

image

Note that the blue Genesis text is clickable and will take you to the following error page:

image

A possible alternative would be the following, with no clickable link:
image

**Review by Andrew47 — CHANGES_REQUESTED** [review]
Overall looks great and tests well across a variety of objects. Would appreciate feedback from @bwann52 on how we handle the Genesis case, otherwise good for merge.

**Comment by Jibz1** [comment]
Screen Shot 2022-06-24 at 12 37 45 PM

@Andrew47 done

**Comment by Andrew47** [comment]
> Screen Shot 2022-06-24 at 12 37 45 PM
> 
> @Andrew47 done

@Jibz1 --> Would it be possible to remove the Last Transaction ID row for Genesis?

**Comment by Jibz1** [comment]
Screen Shot 2022-06-24 at 3 34 47 PM

done

**Review by Andrew47 — APPROVED** [review]
Perfect - good to merge

**Pull request merged**